Makefile

```diff
@@ -79,9 +79,14 @@ core-unit-tests:
 core-unit-tests-jacoco:
 	$(call run-gradle-tasks,$(CORE_MODULES),jacocoTestReport)
 
+.PHONY: core-upload-to-sdk-registry
+core-upload-to-sdk-registry:
+	$(call run-gradle-tasks,$(CORE_MODULES),mapboxSDKRegistryUpload)
+
 .PHONY: core-publish-to-sdk-registry
 core-publish-to-sdk-registry:
-	$(call run-gradle-tasks,$(CORE_MODULES),mapboxSDKRegistryUpload)
+	python3 -m pip install git-pull-request
```

Comment:
Should we add this to our docker image?

Reply:
That'd be great but I believe can be a follow up cc @pengdev

Reply:
Yes, we are still having an ongoing open discussion about the release tools that we use across teams, it would be good to include the agreed release tool to the docker image after the decision is made.